source/bits/lshiftr.ts

```diff
@@ -0,0 +1,21 @@
+import { Bit, Bits } from '../types'
+
+/**
+ * Logical Shift Right
+ * 
+ * Shifts all given bits to right and returns the resulting bits
+ *
+ * @example
+ * lshiftr([1,0,1,1,0,1]) => [0,1,0,1,1,0]
```

Comment:
Not as concerned about this one, but I suppose it’s possible to argue that the leading `0` should be removed.

Reply:
are you sure?